keps/sig-network/1669-proxy-terminating-endpoints/kep.yaml

```diff
@@ -26,7 +26,7 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.22"
```

Comment:
Alternatively, we can try to push the existing feature as-is to Beta in v1,24 which only handle external traffic

Reply:
naah, let's sync it up